login.html

```diff
@@ -1,11 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ko">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css" />
+    <link rel="stylesheet" href="login.css">
     <title>Document</title>
 </head>
 <body>
-    로그인 페이지
+    <container>
+        <div id="logo">
+            <a href="index.html">
+                <img src="./image/판다얼굴.png">
+                <img src="./image/판다마켓.png">
+            </a>    
+        </div>
+        <form>
+            <lable>이메일</lable>
+            <input type="email" name="user-email" placeholder="이메일을 입력해주세요">
+            <lable>비밀번호</lable>
+            <input type="password" name="user-password" placeholder="비밀번호를 입력해주세요">
+            <div class="login-button">
+                로그인
+            </div>
```

Comment:
❗️ 수정요청
해당 태그는 로그인 동작을 수행하는 태그이므로 button 태그가 더 적절합니다.
즉 form에서 제출 이벤트가 발생했을때 해당 form 안의 유저가 입력한 정보를 제출하는 역할을 하므로
button으로 작성해주세요!

```suggestion
            <button type="submit" class="login-button">
                로그인
            </button>
```

button 태그에서 type 속성은 해당 태그의 동작과 연관이 되어 있어 가능하면 늘 기입하시는 것이 좋습니다.
로그인 페이지의 form에서 로그인 버튼, 회원가입 페이지의 form에서 회원가입 버튼과 같은 경우는 submit 타입이고 그외 대부분의 버튼은 button 타입입니다.